test/models.jl

```diff
@@ -0,0 +1,164 @@
+module TestModels
+
+  using SearchLight
+
+  ######## Model from Genie-Searchligth-example-app extracted ############
+  export Book, BookWithInterns
+  using SearchLight, Dates
+
+  ######## Model from Genie-Searchligth-example-app extracted ############
+  export Callback
+  export seed, fields_to_store
+
+  mutable struct Book <: AbstractModel
```

Comment:
The @kwdef phrase is only needed if you want to specify default values by creating instances of a struct. If this isn't needed because you set the values of the fields later you can use it in the form used here. But I know what you mean.